rust/agama-lib/share/profile.schema.json

```diff
@@ -238,16 +271,18 @@
       "type": "object",
       "properties": {
         "language": {
-          "title": "System language ID (e.g., 'en_US')",
-          "type": "string"
+          "title": "System language ID",
+          "type": "string",
+          "examples": ["en_US"]
```

Comment:
We support specifying the whole locale (`en_US.UTF-8`). If no encoding is given (e.g., `en_US`), we fall back to `UTF-8`.